lua/lspconfig/server_configurations/taplo.lua

```diff
@@ -4,9 +4,7 @@ return {
   default_config = {
     cmd = { 'taplo', 'lsp', 'stdio' },
     filetypes = { 'toml' },
-    root_dir = function(fname)
-      return util.root_pattern '*.toml'(fname) or util.find_git_ancestor(fname)
-    end,
+    root_dir = util.find_git_ancestor,
```

Comment:
the docs below still say `root_dir = [[root_pattern("*.toml", ".git")]]`

Reply:
https://github.com/neovim/nvim-lspconfig/pull/3319